.babelrc.json

```diff
@@ -0,0 +1,14 @@
+{
+	"presets": [
+		[
+			"@babel/preset-env", {
+				"loose": true,
+				"modules": "commonjs",
+				"targets": {
+					"node": "6"
+				}
+			}
+		]
+	],
+	"plugins": ["add-module-exports"]
```

Comment:
This was needed because Babel 6 (and thus 7) no longer transform to `module.exports` by default. So in order to [maintain current API](https://www.runpkg.com/?css-modules-loader-core@1.1.0/lib/index.js#67) this plugin was needed.